packages/terra-spacer/docs/README.md

```diff
@@ -0,0 +1,45 @@
+# Terra Spacer
+
+This component is needed to provide a scalar spacing between various components and to help drive a unified spacing system.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-spacer`
+  - `yarn add install terra-spacer`
+
+## Usage
+
+```jsx
+import React from 'react';
+import Spacer from 'terra-spacer';
+
+// Default
+<Spacer >
+    ...
+</Spacer>
+
+// Small view ports
+<Spacer marginTop="small" marginLeft="small-1" paddingTop="small-2">
+  <Image src="someImageSource" alt="example image" />
+<Spacer/>
+
+// Medium view ports
+<Spacer marginLeft="medium" marginRight="medium">
+  <Image src="someImageSource" alt="example image" />
+<Spacer/>
+
+// Large and very large view ports
+<Spacer paddingTop="large" paddingBottom="large+1" paddingLeft="large+2" marginLeft="large+4" marginRight="large+3">
+  <Image src="someImageSource" alt="example image" />
+<Spacer/>
+```
```

Comment:
Can we add an example that implements `bps`?

Reply:
Added https://github.com/cerner/terra-core/pull/1011/commits/51079e4e1f7a6e3bb384bba084478bb7fcf12335#diff-4a98913914f72896bd7b2c1918412f03R55